ecosystem/resources/members/qBraid.toml

```diff
@@ -0,0 +1,17 @@
+name = "qBraid"
+url = "https://github.com/qBraid/qBraid"
+description = "The qBraid-SDK is a Python toolkit for cross-framework abstraction, transpilation, and execution of quantum programs."
+licence = "GNU General Public License (GPL)"
+contact_info = "ryanhill@qbraid.com"
+alternatives = "There are some alternatives to individual modules within the qBraid-SDK e.g. to execute a Qiskit circuit on an AWS device, you could use the qiskit-braket-provider. However, there aren't any other existing projects that support the same matrix of quantum program types and backends through a single, unified interface."
+affiliations = "qBraid Co."
+website = "_No response_"
```

Comment:
```suggestion
```

Reply:
@frankharkins with Eric we relaunch the job to test changes and so the bot push force the bad value for website. Here a code suggestion to delete the field.

Reply:
For the website, it would actually be great if we could add https://qbraid.com. Would that be possible? Alternatively, could add docs page: https://docs.qbraid.com/en/latest/sdk/overview.html. I can make the revision as well

Reply:
I updated the issue :) so next time we run the issue it gonna update the PR :)